modules/nw-metallb-configure-l2-advertisement-interface.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/about-advertising-ipaddresspool.adoc
+
+:_content-type: PROCEDURE
+[id="nw-metallb-configure-with-L2-advertisement-interface_{context}"]
+= Configuring MetalLB with a L2 advertisement by using interface
+
+By default, the IP address from `IPAddressPool` that has been assigned to the service, is advertised from all the network interfaces. The `interfaces` field in the `L2Advertisement` custom resource definitions is used to restrict those network interfaces which advertise the `IPAddressPool`.
```

Comment:
Comma should be removed, and should the 's' be removed on definitions?

```suggestion
By default, the IP address from `IPAddressPool` that has been assigned to the service is advertised from all the network interfaces. The `interfaces` field in the `L2Advertisement` custom resource definition is used to restrict those network interfaces that advertise the `IPAddressPool`.
```

Reply:
Removed 's'.